src/OOVPADatabase/D3D8/5455.inl

```diff
@@ -272,39 +273,22 @@ OOVPA_NO_XREF(D3DDevice_GetTile, 5455, 12)
 // ******************************************************************
 // * D3D::CDevice::KickOff
 // ******************************************************************
-OOVPA_XREF(D3DDevice_KickOff, 5455, 15,
+OOVPA_XREF(D3DDevice_KickOff, 5455, 8,
 
            XREF_D3D_CDevice_KickOff,
            XRefZero)
 {
+    { 0x00, 0x56 },
+
+    { 0x01, 0x8B },
+    { 0x02, 0xF1 },
+
+    { 0x03, 0x8B },
+    { 0x04, 0x46 },
+    { 0x05, 0x08 },
```

Comment:
You should use the `OV_MATCH` macro instead of the old array method. See https://github.com/Cxbx-Reloaded/XbSymbolDatabase/issues/62

Reply:
Also, for ease of reading, consider adding an end of line comment mentioning the disassembled opcode and its arguments

Reply:
revised to OV_MATCH
unable to add opcode comment because OV_MATCH put everything in one line.

Reply:
`OV_MATCH(....); // MOVC ..., ...`

Reply:
One line per opcode

Reply:
interesting, I search the whole database, seems I am the first one to use OV_MATCH.

Reply:
done.